lumos/models/base.py

```diff
@@ -348,10 +326,8 @@ def implicit(
         # FIXME: residuals are not scaled yet
         res = lnp.vector_concat(
             [
-                (model_return.states_dot - states_dot)
-                / self.get_group_scales("states"),
-                (model_return.con_outputs - con_outputs)
-                / self.get_group_scales("con_outputs"),
+                model_return.states_dot - states_dot,
```

Comment:
ideally we'd like to make this dynamic, but for now we could just accept adding a detailed doctoring here to explain the required ordering of outputs from 'implicit'

Reply:
done